pages/docs.html

```diff
@@ -1293,6 +1295,62 @@ <h5>Stacked list</h5>
+            <h3>Rating</h3>
+            <p>
+                Ratings are visual representation of user satisfaction of any product.
+            </p>
+            <div class="component-container rating-container">
+                <div class="rating">
+                    <span class="font-semibold">4.2</span>
+                    <span>
+                        <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" role="img" class="iconify iconify--ri" width="18" height="18" preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24">
+                            <path fill="rgb(145, 55, 135)"
+                                d="m12 18.26l-7.053 3.948l1.575-7.928L.587 8.792l8.027-.952L12 .5l3.386 7.34l8.027.952l-5.935 5.488l1.575 7.928z">
+                            </path>
+                        </svg>
+                    </span>
+                    <span class="font-semibold">|</span>
+                    <span class="font-semibold">5</span>
+                </div>
+                <div class="rating">
+                    <span class="font-semibold">8</span>
+                    <span>
+                        <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" role="img"
+                            class="iconify iconify--ri" width="18" height="18" preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24">
+                            <path fill="rgb(145, 55, 135)"
+                                d="m12 18.26l-7.053 3.948l1.575-7.928L.587 8.792l8.027-.952L12 .5l3.386 7.34l8.027.952l-5.935 5.488l1.575 7.928z">
+                            </path>
+                        </svg>
+                    </span>
+                    <span class="font-semibold">|</span>
+                    <span class="font-semibold">10</span>
+                </div>
+            </div>
+            <div class="code-container">
+                <script type='text/plain' class="line-numbers language-html">
+                    <div class="rating">
+                    <span class="font-semibold">4.2</span>
+                    <span>
+                        <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" role="img" class="iconify iconify--ri" width="18" height="18" preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24">
+                            <path fill="rgb(145, 55, 135)"
+                                d="m12 18.26l-7.053 3.948l1.575-7.928L.587 8.792l8.027-.952L12 .5l3.386 7.34l8.027.952l-5.935 5.488l1.575 7.928z">
+                            </path>
+                        </svg>
+                    </span>
+                    <span class="font-semibold">|</span>
+                    <span class="font-semibold">5</span>
+                </div>
+        
+                </script>
+            </div>
+        
+        </section>
+        <!----------rating-end------------>
```

Comment:
appreciation: good way to comment code so that we can identify respective sections.

Reply:
Thanks Rohit!